docs/src/components/pages/org/org.html

```diff
@@ -0,0 +1,2 @@
+{% render '@page-header--with-icon' %}
```

Comment:
I'd like to know how to use this component but change the title and icon used in the header.
@adborden you might be able to help me solve that?